diff --git a/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py b/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
index 675c070305..40476be9a0 100644
--- a/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
+++ b/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
@@ -153,6 +153,9 @@ def update_libtbx_env(default_dir=None):
         if module.name == 'boost' and os.path.isdir(os.path.join(path, 'boost_adaptbx')):
           new_paths[1] = env.as_relocatable_path(new_path + '_' + module.mate_suffix)
           break
+        if module.name == 'annlib' and os.path.isdir(os.path.join(path, 'annlib_adaptbx')):
+          new_paths[1] = env.as_relocatable_path(new_path + '_' + module.mate_suffix)
+          break
     dist_paths = module.dist_paths
     for i, path in enumerate(dist_paths):
       if path is not None:
