diff --git a/modules/ccp4io/libccp4/ccp4/ccp4_sysdep.h b/modules/ccp4io/libccp4/ccp4/ccp4_sysdep.h
index 3a04df4..7724e25 100644
--- a/modules/ccp4io/libccp4/ccp4/ccp4_sysdep.h
+++ b/modules/ccp4io/libccp4/ccp4/ccp4_sysdep.h
@@ -240,6 +240,9 @@
 # endif
 #endif

+#define NATIVEIT DFNTI_IBO
+#define NATIVEFT DFNTF_LEIEEE
+
 #ifndef NATIVEFT
 #  error "Can't determine machine number format"
 #endif
