diff --git a/modules/cctbx_project/crys3d/hklviewer/hklview_frame.py b/modules/cctbx_project/crys3d/hklviewer/hklview_frame.py
index 99b4a2d5d7..eacaaac5e5 100644
--- a/modules/cctbx_project/crys3d/hklviewer/hklview_frame.py
+++ b/modules/cctbx_project/crys3d/hklviewer/hklview_frame.py
@@ -59,7 +59,7 @@ class HKLViewFrame() :
     self.aniso3 = None
     self.uservectors = []
     self.new_miller_array_operations_lst = []
-    self.copyrightpaths = [("CCTBX copyright", libtbx.env.under_root(os.path.join("modules","cctbx_project","COPYRIGHT.txt"))),
+    self.copyrightpaths = [("CCTBX copyright", libtbx.env.under_build("COPYRIGHT.txt")),
      ("NGL copyright", os.path.join(os.path.dirname(os.path.abspath(__file__)), "License_for_NGL.txt")),
      ("html2canvas copyright", os.path.join(os.path.dirname(os.path.abspath(__file__)), "LICENSE_for_html2canvas.txt"))
     ]
