diff --git a/modules/dxtbx/SConscript b/modules/dxtbx/SConscript
index 8bd47170..d1d3551d 100644
--- a/modules/dxtbx/SConscript
+++ b/modules/dxtbx/SConscript
@@ -97,98 +97,98 @@ else:
     if os.path.exists(psdm_hdf5_path):
         env_etc.dxtbx_hdf5_lib_paths.append(psdm_hdf5_path)

-if not env_etc.no_boost_python and hasattr(env_etc, "boost_adaptbx_include"):
-    Import("env_no_includes_boost_python_ext")
-    env = env_no_includes_boost_python_ext.Clone()
-    env_etc.enable_more_warnings(env=env)
-    env_etc.include_registry.append(
-        env=env,
-        paths=env_etc.dxtbx_includes
-        + env_etc.dxtbx_common_includes
-        + [env_etc.python_include],
-    )
-
-    env.Append(
-        LIBS=env_etc.libm + ["cctbx", "scitbx_boost_python"] + env_etc.dxtbx_libs,
-        LIBPATH=env_etc.dxtbx_lib_paths + env_etc.dxtbx_hdf5_lib_paths,
-    )
-
-    if env_etc.clang_version:
-        wd = ["-Wno-unused-function"]
-        env.Append(CCFLAGS=wd)
-
-    if hasattr(env_etc, "cppdefines"):
-        env.Append(CPPDEFINES=env_etc.cppdefines)
-
-    env.SharedLibrary(
-        target="#lib/dxtbx_ext",
-        source=[
-            "src/dxtbx/boost_python/to_ewald_sphere_helpers.cc",
-            "src/dxtbx/boost_python/ext.cpp",
-            "src/dxtbx/boost_python/compression.cc",
-        ],
-        LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs,
-    )
-
-    nexus = env.SharedLibrary(
-        target="#/lib/dxtbx_format_nexus_ext",
-        source=["src/dxtbx/format/boost_python/nexus_ext.cc"],
-        LIBS=env_etc.libs_python
-        + env_etc.libm
-        + env_etc.dxtbx_libs
-        + env_etc.dxtbx_hdf5_libs,
-    )
-
-    imageset = env.SharedLibrary(
-        target="#/lib/dxtbx_imageset_ext",
-        source=["src/dxtbx/boost_python/imageset_ext.cc"],
-        LIBS=env_etc.libs_python
-        + env_etc.libm
-        + env_etc.dxtbx_libs
-        + env_etc.dxtbx_hdf5_libs,
-    )
-
-    image = env.SharedLibrary(
-        target="#/lib/dxtbx_format_image_ext",
-        source=[
-            "src/dxtbx/format/boost_python/image_ext.cc",
-            "src/dxtbx/format/boost_python/cbf_read_buffer.cpp",
-        ],
-        LIBS=env_etc.libs_python
-        + env_etc.libm
-        + env_etc.dxtbx_libs
-        + env_etc.dxtbx_hdf5_libs,
-    )
-
-    model = env.SharedLibrary(
-        target="#/lib/dxtbx_model_ext",
-        source=[
-            "src/dxtbx/model/boost_python/beam.cc",
-            "src/dxtbx/model/boost_python/spectrum.cc",
-            "src/dxtbx/model/boost_python/goniometer.cc",
-            "src/dxtbx/model/boost_python/kappa_goniometer.cc",
-            "src/dxtbx/model/boost_python/multi_axis_goniometer.cc",
-            "src/dxtbx/model/boost_python/panel.cc",
-            "src/dxtbx/model/boost_python/detector.cc",
-            "src/dxtbx/model/boost_python/scan.cc",
-            "src/dxtbx/model/boost_python/scan_helpers.cc",
-            "src/dxtbx/model/boost_python/crystal.cc",
-            "src/dxtbx/model/boost_python/parallax_correction.cc",
-            "src/dxtbx/model/boost_python/pixel_to_millimeter.cc",
-            "src/dxtbx/model/boost_python/experiment.cc",
-            "src/dxtbx/model/boost_python/experiment_list.cc",
-            "src/dxtbx/model/boost_python/model_ext.cc",
-        ],
-        LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs + env["LIBS"],
-    )
-
-    env.SharedLibrary(
-        target="#/lib/dxtbx_flumpy",
-        source=[
-            "src/dxtbx/boost_python/flumpy.cc",
-        ],
-        LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs + env["LIBS"],
-        CPPFLAGS=["-fvisibility=hidden"],
-    )
-
-    env.SConscript("src/dxtbx/masking/SConscript", exports={"env": env})
+# if not env_etc.no_boost_python and hasattr(env_etc, "boost_adaptbx_include"):
+#     Import("env_no_includes_boost_python_ext")
+#     env = env_no_includes_boost_python_ext.Clone()
+#     env_etc.enable_more_warnings(env=env)
+#     env_etc.include_registry.append(
+#         env=env,
+#         paths=env_etc.dxtbx_includes
+#         + env_etc.dxtbx_common_includes
+#         + [env_etc.python_include],
+#     )
+
+#     env.Append(
+#         LIBS=env_etc.libm + ["cctbx", "scitbx_boost_python"] + env_etc.dxtbx_libs,
+#         LIBPATH=env_etc.dxtbx_lib_paths + env_etc.dxtbx_hdf5_lib_paths,
+#     )
+
+#     if env_etc.clang_version:
+#         wd = ["-Wno-unused-function"]
+#         env.Append(CCFLAGS=wd)
+
+#     if hasattr(env_etc, "cppdefines"):
+#         env.Append(CPPDEFINES=env_etc.cppdefines)
+
+#     env.SharedLibrary(
+#         target="#lib/dxtbx_ext",
+#         source=[
+#             "src/dxtbx/boost_python/to_ewald_sphere_helpers.cc",
+#             "src/dxtbx/boost_python/ext.cpp",
+#             "src/dxtbx/boost_python/compression.cc",
+#         ],
+#         LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs,
+#     )
+
+#     nexus = env.SharedLibrary(
+#         target="#/lib/dxtbx_format_nexus_ext",
+#         source=["src/dxtbx/format/boost_python/nexus_ext.cc"],
+#         LIBS=env_etc.libs_python
+#         + env_etc.libm
+#         + env_etc.dxtbx_libs
+#         + env_etc.dxtbx_hdf5_libs,
+#     )
+
+#     imageset = env.SharedLibrary(
+#         target="#/lib/dxtbx_imageset_ext",
+#         source=["src/dxtbx/boost_python/imageset_ext.cc"],
+#         LIBS=env_etc.libs_python
+#         + env_etc.libm
+#         + env_etc.dxtbx_libs
+#         + env_etc.dxtbx_hdf5_libs,
+#     )
+
+#     image = env.SharedLibrary(
+#         target="#/lib/dxtbx_format_image_ext",
+#         source=[
+#             "src/dxtbx/format/boost_python/image_ext.cc",
+#             "src/dxtbx/format/boost_python/cbf_read_buffer.cpp",
+#         ],
+#         LIBS=env_etc.libs_python
+#         + env_etc.libm
+#         + env_etc.dxtbx_libs
+#         + env_etc.dxtbx_hdf5_libs,
+#     )
+
+#     model = env.SharedLibrary(
+#         target="#/lib/dxtbx_model_ext",
+#         source=[
+#             "src/dxtbx/model/boost_python/beam.cc",
+#             "src/dxtbx/model/boost_python/spectrum.cc",
+#             "src/dxtbx/model/boost_python/goniometer.cc",
+#             "src/dxtbx/model/boost_python/kappa_goniometer.cc",
+#             "src/dxtbx/model/boost_python/multi_axis_goniometer.cc",
+#             "src/dxtbx/model/boost_python/panel.cc",
+#             "src/dxtbx/model/boost_python/detector.cc",
+#             "src/dxtbx/model/boost_python/scan.cc",
+#             "src/dxtbx/model/boost_python/scan_helpers.cc",
+#             "src/dxtbx/model/boost_python/crystal.cc",
+#             "src/dxtbx/model/boost_python/parallax_correction.cc",
+#             "src/dxtbx/model/boost_python/pixel_to_millimeter.cc",
+#             "src/dxtbx/model/boost_python/experiment.cc",
+#             "src/dxtbx/model/boost_python/experiment_list.cc",
+#             "src/dxtbx/model/boost_python/model_ext.cc",
+#         ],
+#         LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs + env["LIBS"],
+#     )
+
+#     env.SharedLibrary(
+#         target="#/lib/dxtbx_flumpy",
+#         source=[
+#             "src/dxtbx/boost_python/flumpy.cc",
+#         ],
+#         LIBS=env_etc.libs_python + env_etc.libm + env_etc.dxtbx_libs + env["LIBS"],
+#         CPPFLAGS=["-fvisibility=hidden"],
+#     )
+
+#     env.SConscript("src/dxtbx/masking/SConscript", exports={"env": env})
diff --git a/modules/dxtbx/libtbx_refresh.py b/modules/dxtbx/libtbx_refresh.py
index fbed593a..1007bbb4 100644
--- a/modules/dxtbx/libtbx_refresh.py
+++ b/modules/dxtbx/libtbx_refresh.py
@@ -35,4 +35,4 @@ def _install_dxtbx_setup():
 if not pkg_resources or any(x.key == "libtbx.dxtbx" for x in pkg_resources.working_set):
     libtbx.pkg_utils.define_entry_points({})

-_install_dxtbx_setup()
+# _install_dxtbx_setup()
