diff --git a/modules/cctbx_project/libtbx/auto_build/bootstrap.py b/modules/cctbx_project/libtbx/auto_build/bootstrap.py
index fd20008e93..6e526f690b 100644
--- a/modules/cctbx_project/libtbx/auto_build/bootstrap.py
+++ b/modules/cctbx_project/libtbx/auto_build/bootstrap.py
@@ -1815,7 +1815,7 @@ environment exists in or is defined by {conda_env}.
       # conda python prefers no environment customizations
       # the get_environment function in ShellCommand updates the environment
       env = {
-        'PYTHONPATH': None,
+        # 'PYTHONPATH': None,
         'LD_LIBRARY_PATH': None,
         'DYLD_LIBRARY_PATH': None
       }
@@ -1850,9 +1850,9 @@ environment exists in or is defined by {conda_env}.
       ))

     # write extra setpaths script for conda
-    if self.use_conda is not None:
-      self.add_command('libtbx.install_conda', args=['--write_setpaths'],
-                       description='Writing additional setup scripts for conda.')
+    # if self.use_conda is not None:
+    #   self.add_command('libtbx.install_conda', args=['--write_setpaths'],
+    #                    description='Writing additional setup scripts for conda.')

   def add_make(self):
     self.add_command('libtbx.scons', args=['-j',
