diff --git a/modules/cctbx_project/boost_adaptbx/SConscript b/modules/cctbx_project/boost_adaptbx/SConscript
index e892dc3450..e760cea33e 100644
--- a/modules/cctbx_project/boost_adaptbx/SConscript
+++ b/modules/cctbx_project/boost_adaptbx/SConscript
@@ -726,7 +726,7 @@ main()
             + u.replace(" ", "_").upper(), file=f)
       print(file=f)
       print("#endif // GUARD", file=f)
-  write_type_id_eq_h()
+  # write_type_id_eq_h()
 
   if (os.path.isfile(libtbx.env.under_build("lib/libswig_class_example.a"))):
     # this example requires these manual steps:
diff --git a/modules/cctbx_project/libtbx/SConscript b/modules/cctbx_project/libtbx/SConscript
index 9f06586b74..95048aa525 100644
--- a/modules/cctbx_project/libtbx/SConscript
+++ b/modules/cctbx_project/libtbx/SConscript
@@ -1149,7 +1149,8 @@ if env_etc.cxx11_is_available:
   test_code += "{i=7; std::cout << i << std::endl;}\n"
   test_code += " x = func(std::vector<int>(2)); return 0;}\n"
   conf = env_base.Configure()
-  flag, output = conf.TryRun(test_code, extension='.cpp')
+  # flag, output = conf.TryRun(test_code, extension='.cpp')
+  flag = True
   conf.Finish()
   if not flag:
     env_etc.cxx11_is_available = False
diff --git a/modules/cctbx_project/libtbx/auto_build/bootstrap.py b/modules/cctbx_project/libtbx/auto_build/bootstrap.py
index 2eaa3ebeab..81f4cfccff 100644
--- a/modules/cctbx_project/libtbx/auto_build/bootstrap.py
+++ b/modules/cctbx_project/libtbx/auto_build/bootstrap.py
@@ -1864,7 +1864,7 @@ environment exists in or is defined by {conda_env}.
       # the get_environment function in ShellCommand updates the environment
       if os.environ.get('CCTBX_CONDA_USE_ENVIRONMENT_VARIABLES', None):
         env = {
-          'PYTHONPATH': None,
+          # 'PYTHONPATH': None,
           'LD_LIBRARY_PATH': None,
           'DYLD_LIBRARY_PATH': None,
           'DYLD_FALLBACK_LIBRARY_PATH': None
@@ -1900,9 +1900,9 @@ environment exists in or is defined by {conda_env}.
       ))
 
     # write extra setpaths script for conda
-    if self.use_conda is not None:
-      self.add_command('libtbx.install_conda', args=['--write_setpaths'],
-                       description='Writing additional setup scripts for conda.')
+    # if self.use_conda is not None:
+    #   self.add_command('libtbx.install_conda', args=['--write_setpaths'],
+    #                    description='Writing additional setup scripts for conda.')
 
   def add_make(self):
     self.add_command('libtbx.scons', args=['-j',
diff --git a/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py b/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
index a3b8594c36..09a2b48197 100644
--- a/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
+++ b/modules/cctbx_project/libtbx/auto_build/conda_build/update_libtbx_env.py
@@ -124,7 +124,7 @@ def update_libtbx_env(default_dir=None):
   for path in sys.path:
     if path.endswith('site-packages'):
       site_packages_path = env.as_relocatable_path(path)
-      break
+      # break
   relocatable_sys_prefix = env.as_relocatable_path(sys_prefix)
   env.repository_paths = [relocatable_sys_prefix, site_packages_path]
   env.scons_dist_path = relocatable_sys_prefix
