diff --git a/modules/lunus/lunus/kokkos/SConscript b/modules/lunus/lunus/kokkos/SConscript
index eba23f2..c2d1cc6 100644
--- a/modules/lunus/lunus/kokkos/SConscript
+++ b/modules/lunus/lunus/kokkos/SConscript
@@ -25,7 +25,7 @@ def detect_architecture(verbose=True):
     else:
       architecture = "HSW"
   except:
-    architecture = "HSW" 
+    architecture = "HSW"
   return architecture
 
 #print("LUNUS: Imported kokkos_env KOKKOS_PATH = {0}\n".format(kokkos_env['KOKKOS_PATH']))
@@ -41,7 +41,7 @@ if os.getenv('LUNUS_KOKKOS_PATH') is None:
   os.environ['LUNUS_KOKKOS_PATH'] = libtbx.env.under_dist('simtbx', '../../kokkos')
 
 if os.environ['LUNUS_KOKKOS_ARCH'] == 'Volta70':
-  kokkos_cxxflags = ['-arch=sm_70','-std=c++14']	
+  kokkos_cxxflags = ['-arch=sm_70','-std=c++14']
 elif os.environ['LUNUS_KOKKOS_ARCH'] == 'Ampere80':
   kokkos_cxxflags = ['-arch=sm_80','-std=c++14']
 else:
@@ -95,10 +95,10 @@ else:
 
 if (env_etc.enable_kokkos):
   kokkos_src = []
-  kokkos_src.append("lmodeim_kokkos.cpp")
+  kokkos_src.append("lmodeim_kokkos.cu")
 
   lunus_kokkos_env.Prepend(CPPPATH=[env_etc.boost_adaptbx_include,
-    env_etc.boost_include])
+    env_etc.boost_include, env_etc.libtbx_include])
 
   lunus_kokkos_env.Append(CPPPATH=[os.path.join(env_etc.lunus_include,"c","include")])
 
@@ -109,4 +109,3 @@ if (env_etc.enable_kokkos):
     source = kokkos_src)
 
   env.Prepend(LIBS=["liblunus_kokkos"])
-
