diff --git a/modules/cctbx_project/smtbx/refinement/tests/tst_least_squares.py b/modules/cctbx_projects/mtbx/refinement/tests/tst_least_squares.py
index 574fc146bf..c711833405 100644
--- a/modules/cctbx_project/smtbx/refinement/tests/tst_least_squares.py
+++ b/modules/cctbx_project/smtbx/refinement/tests/tst_least_squares.py
@@ -23,9 +23,12 @@ from six.moves import range
 tested_ls_engines = (
   least_squares.normal_eqns.non_linear_ls_with_separable_scale_factor_BLAS_2,
 )
-from fast_linalg import env
-if env.initialised:
-  tested_ls_engines += (  least_squares.normal_eqns.non_linear_ls_with_separable_scale_factor_BLAS_3,)
+try:
+  from fast_linalg import env
+  if env.initialised:
+    tested_ls_engines += (  least_squares.normal_eqns.non_linear_ls_with_separable_scale_factor_BLAS_3,)
+except ImportError:
+  print('Skipping fast_linalg checks')
 # we use a wrapper to make sure non_linear_ls_with_separable_scale_factor
 # is not an argument with a default value, so as to protect ourself from
 # forgetting to specify that argument in the tests, which would result
